groups/rds/data.tf

```diff
@@ -6,6 +6,7 @@ data "vault_generic_secret" "secrets" {
   path = "team-${var.team}/${var.account_name}/${var.region}/${var.environment}/${var.repository_name}"
 }
 
+data "aws_region" "current" {}
```

Comment:
This data resource doesn't appear to be used anywhere else - is it required?

Reply:
Does not be not be used by anywhere. Removed from both instance & RDS groups